docs/programming/kotlin/fundamentals/functions.md

```diff
@@ -0,0 +1,11 @@
+# Functions
+
+## Declaring a Function
+
+### Using Functions
+
+### Arguments & Parameters
+
+## Return Types
+
+
```

Comment:
Don't know whether getter/setter functions would fall under here or variables. Let me know where you plan to put it (it's something that's actually quite useful imo.)